nixos/tests/pam-oath-login.nix

```diff
@@ -1,4 +1,4 @@
-import ./make-test.nix ({ ... }:
```

Comment:
This should go in a separate PR.

Reply:
Well, at least it's 2 separate commits, so we can merge this as is.

Still, please open 2 separate PRs next time.

Reply:
Well, at least it's 2 separate commits, so we can merge this as is.

Still, please open 2 separate PRs next time.